tests/installation/partitioning_raid.pm

```diff
@@ -60,9 +61,14 @@ sub addpart {
     send_key $cmd{donotformat};
     send_key "tab";
     send_key 'alt-i' if is_storage_ng;    # Select file system
+    my $partition_type;
```

Comment:
variable never used

Reply:
Nice catch, that's remains of on if the approaches, thanks!